languages.toml

```diff
@@ -12,10 +12,6 @@ extensions = ["ncl"]
 
 [[language]]
 name = "ocaml"
-extensions = ["ml", "mli"]
-
-[[language]]
-name = "ocaml_implementation"
 extensions = ["ml"]
```

Comment:
Yes, it can, because now we have this:

```
[[language]]
name = "ocaml"
extensions = ["ml"]

[[language]]
name = "ocaml_interface"
extensions = ["mli"]
```